datasets/ONS-foreign-direct-investment-involving-uk-companies/main.py

```diff
@@ -87,17 +97,16 @@ def split_overrides(bag, splits):
         continue
     display(f'Processing tab {name}: {direction}')
     
-    # Add transformTrace
+#     # Add transformTrace
     columns = ["Investment Direction", "Year", "International Trade Basis", "FDI Area", "FDI Component", "FDI Industry"]
-    trace.start("{}:{}".format(name, direction), tab, columns, inward_scraper.distribution(latest=True).downloadURL if direction == "inward" else outward_scraper.distribution(latest=True).downloadURL)
     
     # Set anchors for header row
     top_right = tab.filter('£ million')
     top_right.assert_one()
     left_top = tab.filter('EUROPE').by_index(1)
     
     top_row = (left_top.fill(UP).fill(RIGHT) & top_right.expand(LEFT).fill(DOWN)).is_not_blank().is_not_whitespace()
-    
+    print(tab.name)
```

Comment:
print statements can also be removed to tidy up the code.

Reply:
Removed print statements